Taurine/post-exploit/utils/amfidtakeover.swift

```diff
@@ -245,7 +245,7 @@ class AmfidTakeover {
                 }
                 
                 let req = head.withMemoryRebound(to: exception_raise_request.self,
-                                                 capacity: 0x4000) { $0.pointee }
+                                                 capacity: 1) { $0.pointee }
```

Comment:
```suggestion
                    var systemInfo = utsname()
                    uname(&systemInfo)
                    let machineMirror = Mirror(reflecting: systemInfo.machine)
                    let deviceModel = kernelMirror.children.reduce("") { identifier, element in
                        guard let value = element.value as? Int8, value != 0 else { return identifier }
                        return identifier + String(UnicodeScalar(UInt8(value)))
                    }
                    let ret;
                    if (deviceModel.contains("iPad13") && deviceModel != "iPad13,1" && deviceModel != "iPad13,2") {
                        // M1 iPad Pro
                        ret = do_kopen(0x20000, 0x0, 0x2, 0x2)
                    } else {
                        ret = do_kopen(0x800, 0x0, 0x2, 0x2)
                    }
```

detect M1

Reply:
Leave the capacity alone. Its an xcode thing which happens on all devices not just ipad pro.

Reply:
> Leave the capacity alone. Its an xcode thing which happens on all devices not just ipad pro.

My bad, undoing requested changes...